README.md

```diff
@@ -298,7 +298,7 @@ Sooner or later I will complete it with the relative answers. Feel free to contr
 * Explain Unicode/Database Transactions to a 5 year old child.
 * Defend the monolithic architecture.
 * What does it mean to be a "Professional Developer"?
-* Is developing software an art, a craftsmanship or an engineering endeavour? Your opinion.
+* Is developing software an art, a craftsmanship or an engineering endeavor? Your opinion.
```

Comment:
Again, UK vs US spelling 🙂

Reply:
(Jokingly) -- it seems like the solution is for a en(GB) branch and an en(US) branch!